charts/mongodb/remedy.md

```diff
@@ -0,0 +1,90 @@
+## TODOs on MongoDB Critical Alerts
+
+### Database Alerts
+
+- #### MongoDBDown
+  - Describe the `MongoDB` CR, check the reason from conditions and try restarting the pods
+  - Check database pod logs
+  - Contact AppsCode team
+- #### MongodbReplicationLag
+  - Check if there is any network issue which is causing the replication lag.
+- #### MongodbTooManyConnections
+  - Increase `net.maxIncomingConnections` value using [MongoDBOpsRequest](https://kubedb.com/docs/latest/guides/mongodb/reconfigure/overview/)
+- #### MongodbVirtualMemoryUsage
+  - Increase storage and/or memory using [VerticalScaling OpsRequest](https://kubedb.com/docs/latest/guides/mongodb/scaling/vertical-scaling/overview/) and [VolumeExpansion OpsRequest](https://kubedb.com/docs/latest/guides/mongodb/volume-expansion/overview/)
+- #### MongoDBRecurrentMemoryPageFaults
+  - Increase Memory using [VerticalScaling OpsRequest](https://kubedb.com/docs/latest/guides/mongodb/scaling/vertical-scaling/overview/)
+  - Index your data properly
+- #### MongodbNumberCursorsOpen
+  - Check for slow queries using `db.currentOp()` command
+  - Check if application that is using mongodb is closing cursors properly
+- #### MongoDBHighLatency | MongodbCursorsTimeouts | MongoDBRecurrentCursorTimeout | MongoDBHighTicketUtilization
+  - Check for slow queries using `db.currentOp()` command
+  - Check database pod logs
+  - Check for network and disk performance
+  - Contact AppsCode team if this issue persists for a long time
```

Comment:
"for long time" is not definitive.  Using "for more than 1 hour" or something like that, would be better.